### DISCUSSION TITLE
grott docker container keeps restarting for ever.

### DISCUSSION THREAD

**Discussion by GroenZoom** [johanmeijer/grott#371] [General]
Hallo,
 I am running HomeAssistant Supervised 2023.7.2  on RPI 3B+ and trying to get Growatt PV data via Grott to MQTT. 
The latest docker image AMD64 ledidobe/grott is pulled in and can be run but keeps contineously restarting.
Nothing happens in MQTT, no verbose errors and I can,'t find a log with hints. Only Putty sudo docker ps shows it.
Make me happy with a solution or suggestion and I will be gratefull forever.
Regards Martin

**Comment by johanmeijer** [comment]
I supose we can close this one (same as #372?)

**Comment by GroenZoom** [comment]
Yes, i am happy that it working now anz looking to reduce data push interval down to 1min.Thanks and regards.MartinOp 26 jul. 2023 21:09 schreef Johan Meijer ***@***.***>:
I supose we can close this one (same as #372?)
